src/types/message/index.js

```diff
@@ -154,11 +195,17 @@ class BitswapMessage {
     return Message.encode(msg)
   }
 
+  /**
+   * @param {BitswapMessage} other
+   * @returns {boolean}
+   */
   equals (other) {
     if (this.full !== other.full ||
         this.pendingBytes !== other.pendingBytes ||
         !isMapEqual(this.wantlist, other.wantlist) ||
         !isMapEqual(this.blocks, other.blocks) ||
+        // @TODO - Is this a bug ?
+        // @ts-expect-error - isMap equals map values to be objects not numbers
```

Comment:
@hugomrdias This looks like a bug to me, could you please confirm or tell me if I'm not seeing something ?

Reply:
Looks like a bug